spyderlib/plugins/shortcuts.py

```diff
@@ -354,12 +361,66 @@ def setup_page(self):
         reset_btn = QPushButton(_("Reset to default values"))
         reset_btn.clicked.connect(self.reset_to_default)
         vlayout.addWidget(reset_btn)
+        self.emacs_cb = self._get_emacs_cb()
+        self.emacs_cb.clicked.connect(self.emacs_checked)
+        vlayout.addWidget(self.emacs_cb)
         self.setLayout(vlayout)
+
+    def _get_emacs_cb(self):
+        cb = QCheckBox('Use Emacs Keybindings')

```

Comment:
I like this!! You read my mind, that was what I was thinking: to add a checkbox for Emacs :-)